tests/integration/targets/azure_rm_monitoractiongroups/tasks/main.yml

```diff
@@ -0,0 +1,362 @@
+    resource_group: "{{ resource_group }}"
+    name: "actiongroup-{{ random_string }}"
+    state: present
+    enabled: true
+    location: Global
+    email_receivers:
+      - email_address: "{{ action_group_mail }}"
+        name: "sendmailto{{ action_group_mail | split('@') | first }}"
+        use_common_alert_schema: true
+    group_short_name: AGShortName
+  register: output
+
+- name: Assert that action group has not changed during idempotency check (no tags)
+  ansible.builtin.assert:
+    that:
+      - not output.changed
+
+- name: Create action group (creation tags)
+  azure.azcollection.azure_rm_monitoractiongroups:
+    resource_group: "{{ resource_group }}"
+    name: "actiongroup-creationtags-{{ random_string }}"
+    state: present
+    enabled: true
+    location: Global
+    email_receivers:
+      - email_address: "{{ action_group_mail }}"
+        name: "sendmailto{{ action_group_mail | split('@') | first }}"
+        use_common_alert_schema: true
+    group_short_name: AGShortName
+    tags:
+      CreationTag1: CreationTagValue1
+  register: output
+
+- name: Assert that action group has been created (no tags)
+  ansible.builtin.assert:
+    that:
+      - output.changed
+      - output.actiongroup.tags.CreationTag1 == 'CreationTagValue1'
+
+- name: Create action group (creation tags, idempotency check)
+  azure.azcollection.azure_rm_monitoractiongroups:
+    resource_group: "{{ resource_group }}"
+    name: "actiongroup-creationtags-{{ random_string }}"
+    state: present
+    enabled: true
+    location: Global
+    email_receivers:
+      - email_address: "{{ action_group_mail }}"
+        name: "sendmailto{{ action_group_mail | split('@') | first }}"
+        use_common_alert_schema: true
+    group_short_name: AGShortName
+    tags:
+      CreationTag1: CreationTagValue1
+  register: output
+
+- name: Assert that action group has not changed during idempotency check (creation tags)
+  ansible.builtin.assert:
+    that:
+      - not output.changed
+      - output.actiongroup.tags.CreationTag1 == 'CreationTagValue1'
+
+- name: Add tag to existing action group
+  azure.azcollection.azure_rm_monitoractiongroups:
+    resource_group: "{{ resource_group }}"
+    name: "actiongroup-{{ random_string }}"
+    state: present
+    tags:
+      TestTag1: TestValue1
+  register: output
+
+- name: Assert that action group has been changed (add tag)
+  ansible.builtin.assert:
+    that:
+      - output.changed
+      - output.actiongroup.tags.TestTag1 == 'TestValue1'
+
+- name: Add tag to existing action group (idempontency)
+  azure.azcollection.azure_rm_monitoractiongroups:
+    resource_group: "{{ resource_group }}"
+    name: "actiongroup-{{ random_string }}"
+    state: present
+    tags:
+      TestTag1: TestValue1
+  register: output
+
+- name: Assert that action group has been changed (add tag, idempotency)
+  ansible.builtin.assert:
+    that:
+      - not output.changed
```

Comment:
```suggestion
      - output is changed
```

Reply:
this is changing the meaning, Ihave to revert it, also some others :) but I'll fix it in the spirit of your change :)